### PR TITLE
add image no content

### PR DESCRIPTION
**Pull request by BohdanBusko**
**STATUS**
READY

**DESCRIPTION**
screenshot from 2018-11-16 14-18-06

**CHECK LIST**
- [ ] specs added
- [ ] specs passed
- [ ] code coverage >90%
- [ ] DB indexes added if needed
- [ ] seed.rb file is updated if needed
- [ ] PR is reviewed manually again (to make sure you have 100% ready code)
- [ ] PR doesn't have conflicts with main branch
- [ ] I've checked new feature as logged in and logged out user if needed
- [ ] "let!" is used in specs

### CODE REVIEW

File: app/views/account/projects/show.html.slim [hunk 1]
```diff
@@ -10,21 +10,22 @@
         = link_to "Add Task", new_account_project_task_path(@project), class: "button is-link"
       .level-item
         = link_to "Add Section", new_account_project_task_path(@project, section: true), class: "button is-link"
+  - if @project.tasks.exists?
```
Comment (romaslmd) [inline app/views/account/projects/show.html.slim]: yet the same mistake

File: app/views/account/projects/show.html.slim [hunk 2]
```diff
@@ -10,21 +10,22 @@
...
-        = render '/account/projects/show_tasks', task: task, project: @project
-
+    .conteiner.box
+      .title
+        |  Сompleted Tasks:
+      #complete_tasks
+        - @complete_tasks.each do |task|
+          = render '/account/projects/show_tasks', task: task, project: @project
+  - else
+    = image_tag('undraw_no_data_qbuo.svg', class: 'image no-content')
```
Comment (romaslmd) [inline app/views/account/projects/show.html.slim]: why no content is on the image?

File: app/assets/images/undraw_no_data_qbuo.svg [hunk 3]
```diff
@@ -0,0 +1 @@
...-.51-.1h0a2.31,2.31,0,0,0-1.53,4.15h0a2.22,2.22,0,0,0,.45.26,4.61,4.61,0,0,1,2.57,5.57,2.22,2.22,0,0,0-.1.51h0a2.31,2.31,0,0,0,4.15,1.53h0a2.22,2.22,0,0,0,.26-.45A4.61,4.61,0,0,1,325,631.4a2.22,2.22,0,0,0,.51.1h0a2.31,2.31,0,0,0,1.53-4.15h0A2.22,2.22,0,0,0,326.59,627.09Z" transform="translate(-189.92 -59.59)" fill="#fdd835" opacity="0.5"/><path d="M855,127.77a4.61,4.61,0,0,1-2.57-5.57,2.22,2.22,0,0,0,.1-.51h0a2.31,2.31,0,0,0-4.15-1.53h0a2.22,2.22,0,0,0-.26.45,4.61,4.61,0,0,1-5.57,2.57,2.22,2.22,0,0,0-.51-.1h0a2.31,2.31,0,0,0-1.53,4.15h0a2.22,2.22,0,0,0,.45.26,4.61,4.61,0,0,1,2.57,5.57,2.22,2.22,0,0,0-.1.51h0a2.31,2.31,0,0,0,4.15,1.53h0a2.22,2.22,0,0,0,.26-.45,4.61,4.61,0,0,1,5.57-2.57,2.22,2.22,0,0,0,.51.1h0a2.31,2.31,0,0,0,1.53-4.15h0A2.22,2.22,0,0,0,855,127.77Z" transform="translate(-189.92 -59.59)" fill="#fdd835" opacity="0.5"/><circle cx="812.64" cy="314.47" r="7.53" fill="#f55f44" opacity="0.5"/><circle cx="230.73" cy="746.65" r="7.53" fill="#f55f44" opacity="0.5"/><circle cx="735.31" cy="477.23" r="7.53" fill="#f55f44" opacity="0.5"/><circle cx="87.14" cy="96.35" r="7.53" fill="#4d8af0" opacity="0.5"/><circle cx="7.53" cy="301.76" r="7.53" fill="#47e6b1" opacity="0.5"/></svg>
```
Comment (romaslmd) [inline app/assets/images/undraw_no_data_qbuo.svg:1]: why id?

Comment (BohdanBusko) [reply app/assets/images/undraw_no_data_qbuo.svg:1]: це сам image_tag генерить

**Review by igorkasyanchuk — APPROVED** [review]

**Pull request merged**